src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
@JunTaoLuo - Can we move this out of the Targets folder into a separate folder called "DefaultContent" or something similar. This folder is meant for all msbuild targets.

Reply:
Sure, that's actually what I had originally, but I figured I'll keep everything together until someone tells me otherwise 😄

Reply:
wish granted :)

Reply:
File removed.